ButteryToast/Toast.swift

```diff
@@ -14,25 +14,32 @@ import UIKit
  This allows for managing toasts in a queue and determining if they have been presented yet.
  */
 
+public enum Orientation {
+  case top
+  case bottom
+}
+
 open class Toast: Operation {
   
   var duration: TimeInterval
   var delay: TimeInterval
-
+  var orientation: Orientation
+  
   weak var toaster: Toaster?
   
   private let view: UIView
   private let transitionDuration: TimeInterval = 0.3
-
+  
   /**
    Initializes the `Toast` instance with the specified view.
    - parameter dismissAfter: If set, the time interval after which a Toast will auto-dismiss without user interaction. If unset, a Toast will persist until dismissed.
    - parameter height: If set, the height of the toast, enforced by AutoLayout. If unset, the height will be determined soley from the intrinsic content size of the view passed to the Toast.
-  */
-  public init(view: UIView, duration: TimeInterval = 2.0, delay: TimeInterval = 0.0) {
+   */
+  public init(view: UIView, duration: TimeInterval = 2.0, delay: TimeInterval = 0.0, orientation: Orientation) {
```

Comment:
just thinking about use cases, I think it will be better if Orientation doesn't have to be decided each time you want to present a toast. Especially since you don't know at time of creation when a toast will be presented. Instead, how about setting on the Toaster, and have the toast determine it's presentation from the Toaster at time of presentation? (in `main`)

Reply:
Really like the use of an `enum` btw 👍

Reply:
actually how about this - add the ability to set default orientation on the Toaster but can optionally specify on a per-toast basis. just check if an orientation was set on the Toast in `main`, then fall back to Toaster default if not.